system/Cache/Handlers/PredisHandler.php

```diff
@@ -28,39 +29,99 @@ class PredisHandler extends BaseHandler
      * @var array
      */
     protected $config = [
-        'scheme'   => 'tcp',
-        'host'     => '127.0.0.1',
-        'password' => null,
-        'port'     => 6379,
-        'timeout'  => 0,
+        'host'       => '127.0.0.1',
+        'username'   => null,
+        'password'   => null,
+        'port'       => 6379,
+        'timeout'    => 0,
+        'database'   => 0,
+        'isCluster'  => false,
+        'persistent' => false,
     ];
 
     /**
      * Predis connection
      *
-     * @var Client
+     * @var Client|null
```

Comment:
This shuts up phpstan errors for whenever we call `isset($this->redis)`.  `$redis` WILL be null until we call `initialize()`, so it's valid to check if it's set or not.